cluster/node-pools/worker-default/userdata.clc.yaml

```diff
@@ -22,6 +22,24 @@ systemd:
   - name: locksmithd.service
     mask: true
 
+  # this should only be enabled for instances with instance storage. e.g.
+  # i3.large instances
+  {{if index .NodePool.ConfigItems "instance_storage"}}
+  - name: data-disk0.mount
+    enable: true
+    contents: |
+      [Unit]
+      Before=local-fs.target
+
+      [Mount]
+      What=/dev/nvme0n1
```

Comment:
The path should also be automatically determined based on whether the instance is NVMe-based  or not.